sydney/sydney.py

```diff
@@ -173,18 +208,30 @@ async def _ask(
         ):
             raise NoConnectionException("No connection to Bing Chat was found")
 
+        blob_data = {
```

Comment:
I think this entire piece of code would fit better inside the `_build_ask_arguments` method, as it is part of building all the arguments that we're sending